docs/contracts.rst

```diff
@@ -1023,23 +1048,33 @@ When the inheritance results in a contract with a function and a modifier of the
 This error is produced also by an event and a modifier of the same name, and a function and an event of the same name.
 As an exception, a state variable getter can override a public function.
 
+.. index:: ! abstractions
+
+.. _abstractions:
+
+************
+Abstractions
+************
+
+An abstraction type describes a contract but does not provide a full implementation of the contract. An abstraction is implemented as abstract contract or interface.
```

Comment:
Not sure what "abstraction type" means here.

Reply:
I see where the confusion comes from, since there's no actual abstract type in Solidity it's probably best to rename "abstraction type" to just "abstraction"